core_impl/src/test/java/com/google/instrumentation/stats/StatsContextFactoryTest.java

```diff
@@ -29,14 +31,19 @@
  */
 @RunWith(JUnit4.class)
 public class StatsContextFactoryTest {
```

Comment:
> Since StatsSerializer.serialize() will only be called by StatsContextImpl.serialize(), tests for serialize() will be under StatsContextTest (https://github.com/google/instrumentation-java/blob/master/core_impl/src/test/java/com/google/instrumentation/stats/StatsContextTest.java#L143).

The tests that are currently in StatsContextTest.java are all round-trip tests, so they aren't testing the encoding.  I meant that it would be nice to have some tests that create a StatsContext, serialize it, and compare it against a byte[] with the expected encoding.  They could also go in StatsContextTest.java.

> I'll add tests for mixed tags.

I meant that we should have a test for multiple String tags, since we can't test the other types yet.